playground/infrastructure/config.py

```diff
@@ -26,4 +28,12 @@ class Config:
     SERVER_ADDRESS = os.getenv("SERVER_ADDRESS", "localhost:8080")
     ERROR_STATUSES = [STATUS_VALIDATION_ERROR, STATUS_ERROR, STATUS_PREPARATION_ERROR, STATUS_COMPILE_ERROR,
                       STATUS_RUN_TIMEOUT, STATUS_RUN_ERROR]
-    SUPPORTED_SDK = {'java': SDK_JAVA}
+    SUPPORTED_SDK = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON}
+
+
+@dataclass(frozen=True)
+class TagFields:
+    NAME: str = "name"
+    DESCRIPTION: str = "description"
+    MULTIFILE: bool = "multifile"
+    CATEGORIES: List[str] = "categories"
```

Comment:
One more thing, if we would like to add default values anyway, then the corresponding default values should be added. I.e. instead of `MULTIFILE: bool = "multifile"` use `MULTIFILE: bool = False`

Reply:
Changed.